src/Flyout/Flyout.js

```diff
@@ -52,5 +52,5 @@ Flyout.propTypes = {
   idealDirection: PropTypes.oneOf(['up', 'right', 'down', 'left']),
   onDismiss: PropTypes.func.isRequired,
   positionRelativeToAnchor: PropTypes.bool,
-  size: PropTypes.oneOf(['xs', 'sm', 'md', 'lg', 'xl']), // default: sm
+  size: PropTypes.oneOf(['xs', 'sm', 'md', 'lg', 'xl', PropTypes.number]), // default: sm
```

Comment:
I don't think this will do what you want it to. It's looking for the exact usage of `PropTypes.number` rather than

```
PropTypes.oneOfType(PropTypes.number, PropTypes.oneOf([ ... ]))
```